processor/src/tasks/processGerbers/index.ts

```diff
@@ -113,35 +113,29 @@ export default async function processGerbers(
         job.updateProgress({ status: 'failed', file: topSvgPath, error }),
       )
 
-    promises.push(
-      generateTopPng(topSvgPath, stackup, topPngPath)
-        .then(() => job.updateProgress({ status: 'done', file: topPngPath }))
-        .catch(error =>
-          job.updateProgress({ status: 'failed', file: topPngPath, error }),
-        ),
-    )
+    await generateTopPng(topSvgPath, stackup, topPngPath)
+      .then(() => job.updateProgress({ status: 'done', file: topPngPath }))
+      .catch(error =>
+        job.updateProgress({ status: 'failed', file: topPngPath, error }),
+      )
 
-    promises.push(
-      generateTopLargePng(topSvgPath, stackup, topLargePngPath)
-        .then(() => job.updateProgress({ status: 'done', file: topLargePngPath }))
-        .catch(error =>
-          job.updateProgress({ status: 'failed', file: topLargePngPath, error }),
-        ),
-    )
+    await generateTopLargePng(topSvgPath, stackup, topLargePngPath)
+      .then(() => job.updateProgress({ status: 'done', file: topLargePngPath }))
+      .catch(error =>
+        job.updateProgress({ status: 'failed', file: topLargePngPath, error }),
+      )
 
     await generateTopMetaPng(topSvgPath, stackup, topMetaPngPath)
       .then(() => job.updateProgress({ status: 'done', file: topMetaPngPath }))
       .catch(error =>
         job.updateProgress({ status: 'failed', file: topMetaPngPath, error }),
       )
 
-    promises.push(
-      generateTopWithBgnd(topMetaPngPath, topWithBgndPath)
-        .then(() => job.updateProgress({ status: 'done', file: topWithBgndPath }))
-        .catch(error =>
-          job.updateProgress({ status: 'failed', file: topWithBgndPath, error }),
-        ),
-    )
+    await generateTopWithBgnd(topMetaPngPath, topWithBgndPath)
+      .then(() => job.updateProgress({ status: 'done', file: topWithBgndPath }))
+      .catch(error =>
+        job.updateProgress({ status: 'failed', file: topWithBgndPath, error }),
+      )
```

Comment:
> Removes concurrency of inkscape executions which take up a lot of memory

I guess this is the code for this change, a comment would be helpful to say these are done sequentially or even better group them into a task.